server/routes.spec.ts

```diff
@@ -1,46 +1,317 @@
 import { InitializeEnv } from './test_data/initialize_env';
-
 import { logger } from '@project-sunbird/ext-framework-server/logger';
-import * as express from 'express';
-import * as bodyParser from 'body-parser';
 import * as _ from "lodash";
 import * as supertest from 'supertest';
 import { ConnectToServer } from './test_data/routes.test.server';
 import { expect } from 'chai';
+import { telemetry_v1, telemetry_v3, registerDevice } from './test_data/routes.spec.data';
 
 
 const initialzeEnv = new InitializeEnv();
 let server = new ConnectToServer();
+let app;
+initialzeEnv.init();
 
-describe('Routes', () => {
-    let app;
-    initialzeEnv.init();
-    before('StartServer', async () => {
-            await server.startServer().then(res => {
-                app = res;
-                logger.info(`Server Connected`);
-            }).catch(err => {
-                logger.error(`Received Error while connecting to server err: ${err}`);
-            });
+before('StartServer', async () => {
+    await server.startServer().then(res => {
+        app = res;
+        logger.info(`Server Connected`);
+    }).catch(err => {
+        logger.error(`Received Error while connecting to server err: ${err}`);
     });
+});
+
+
 
-    it('resourcebundles', (done) => {
+describe('All', () => {
+
+    it('All', (done) => {
+        supertest(app)
+            .get('/')
+            .set('Content-Type', 'text/html; charset=utf-8')
+            .expect(200)
+            .end((err, res) => {
+                if (res.statusCode >= 500) { logger.error(err); return done(); }
+                if (err && res.stausCode >= 400) { expect.fail(); return done(err); };
+                done();
+            });
+    }).timeout(20000);
+})
+
+
+describe('Test Resourcebundle', () => {
+
+    it('#resourcebundle', (done) => {
         supertest(app)
             .get(`/resourcebundles/v1/read/en`)
             .expect('Content-Type', 'application/json; charset=utf-8')
             .expect(200)
             .end((err, res) => {
-                if (err) throw err;
-                expect(res.body.result.id).to.equal('api.resoucebundles.read').and.string;
-                expect(res.body.result.ver).to.equal('1.0').and.string;
+                if (res.statusCode >= 500) { logger.error(err); return done(); }
+                if (err && res.stausCode >= 400) { expect.fail(); return done(err); };
+                expect(res.body.responseCode).to.equal('OK');
+                expect(res.body.id).to.equal('api.resoucebundles.read').to.be.a('string');
```

Comment:
check one string value should be in english and add another test case to check for another language to be that Example:  expect(res.body.result.result.consumption.t12).to.be('play')
for other language expect(res.body.result.result.consumption.t12).to.be('క్రీడా')